src/allocators/types/TribunalStructs.sol

```diff
@@ -18,15 +18,55 @@ struct BatchClaim {
     bytes allocatorSignature; // Authorization from the allocator
 }
 
+// struct Mandate {
```

Comment:
shouldn't we just leverage the structs in the tribunal repo as a lib rather than redefine them here?